forward/lib/forward/forward_id.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:equatable/equatable.dart';
+import 'package:forward/forward/converter/forward_id_coverter.dart';
+import 'package:jmap_dart_client/http/converter/id_converter.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+
+enum ForwardIdType {
```

Comment:
IMO, just creating a `factory` for `ForwardId` is enough. No need to create `ForwardIdType` anymore. Ex:
```dart
factory ForwardId.singleton() {
    return ForwardId(Id('singleton'));
}
```

What do you guys mean? @hoangdat @ManhNTX

Reply:
I think create enum type make it clear and easy maintain and edit later

Reply:
oke